### ISSUE TITLE
Is simulator connectable to your service?

### ISSUE THREAD

**Issue by jay0129** [DeviceFarmer/stf_ios_support#70]
I have failed trying to run stf-ios-provider.
I will try from the beginning with iOS simulator with my new PC.
But before starting my work, is it possible to use simulator?

**Comment by nanoscopic** [comment]
The simulator is not supported right now for a few reasons:
1. The video streaming methods currently in place won't work with it
2. The code is specifically setup to only activate on connection of a real USB device

The second part is relatively easy to fix... The first is harder as it requires a compatible streaming method.

I have developed a new method of video streaming that I believe should work, using the "broadcast upload extension". I have yet to test it with a simulator. It will ultimately be an app that I will release on the app store to do video streaming.

Essentially: I think I can make simulator work in the near future. I'll look into it and update this with more information within the next few weeks.

**Comment by nanoscopic** [comment]
Also, this is a duplicate of #6.
